runtime/src/launch/gasnetrun_common/gasnetrun_common.h

```diff
@@ -36,11 +36,12 @@ static char _nlbuf[16];
 static char** chpl_launch_create_argv(const char *launch_cmd,
                                       int argc, char* argv[],
                                       int32_t numLocales) {
-  const int largc = 7;
-  char *largv[largc];
+  const int maxlargc = 7 + 4;
```

Comment:
Little weird to not just have 11. Or maybe make these names variables and use them to update largc?

Reply:
11 is fine.  I'll do that.

Reply:
I ended up going with the named quantity instead, to make it easier to move the code into a central place.